.github/workflows/security.yml

```
@@ -263,50 +449,473 @@ jobs:
       - name: 🚨 Create critical security issue
-        if: (needs.security-deep-scan.outputs.critical-vulnerabilities > 0 || needs.dependency-check.outputs.dependency-vulnerabilities > 0) && (github.event.inputs.create_issue == 'true' || github.event.inputs.create_issue == '')
+        if: (needs.security-deep-scan.outputs.critical-vulnerabilities > 0 || needs.dependency-check.outputs.critical-vuln-count > 0 || needs.security-deep-scan.outputs.secrets-found > 0) && (github.event.inputs.create_issue == 'true' || github.event.inputs.create_issue == '')
         uses: actions/github-script@v7
+        env:
+          EXECUTIVE_SUMMARY: ${{ env.EXECUTIVE_SUMMARY }}
+          SECRETS_DETAILS: ${{ env.SECRETS_DETAILS }}
+          DEPENDENCIES_DETAILS: ${{ env.DEPENDENCIES_DETAILS }}
+          CODE_VULNERABILITIES_DETAILS: ${{ env.CODE_VULNERABILITIES_DETAILS }}
+          ENV_FILES_DETAILS: ${{ env.ENV_FILES_DETAILS }}
         with:
           script: |
```

actionlint (1.7.7)

property "critical-vuln-count" is not defined in object type {dependency-vulnerabilities: string}

(expression)